File: loops/fizzbuzz.c
// Anthony Petersen, FizzBuzz
#include <stdio.h>
int number = 1;


int main(void){
    while(number <= 50){
        if(number%3==0 && number%5==0){
            printf("FizzBuzz\n");
        }else if(number%5==0){
            printf("Buzz\n");
        }else if(number%3==0){
            printf("Fizz\n");
        }else{
            printf("%d\n", number);
        }
            number += 1;
        }

return 0;
}

File: Loops/fizzbuzz.py
#Anthony Petersen, FizzBuzz
number = 1
while number <= 50:
    if number % 3 == 0 and number % 5 == 0:
        print("FizzBuzz")
    elif number % 5 == 0:
        print("Buzz")
    elif number % 3 ==0:
        print("Fizz")
    else:
        print(number)
    number += 1